staging/src/k8s.io/kube-aggregator/pkg/apiserver/apiservice_controller.go

```diff
@@ -96,7 +97,10 @@ func (c *APIServiceRegistrationController) sync(key string) error {
 		return err
 	}
 
-	c.apiHandlerManager.AddAPIService(apiService, c.getDestinationHost(apiService))
+	host := c.getDestinationHost(apiService)
+	if len(host) > 0 {
```

Comment:
If an APIService is updated to remove its referenced service (becomes local), we need to update to reflect that, right?